docs/operator-manual/airgap/01-requirements.md

```diff
@@ -0,0 +1,10 @@
+---
+sidebar_label: "Requirements"
+title: ""
+---
+
+# Requirements
+
+1. Private registry that supports OCI artifacts. It will be used for storing the container images and policies.
```

Comment:
We need to give some reference names and versions. I think we already have that inside of our docs, don't we? If that's the case we can just add a link pointing to that.

Reply:
added in https://github.com/kubewarden/docs/pull/150/commits/59e2987985857ff1b6036e06707d6a1dceeee2ae